src/org/usfirst/frc/team467/robot/Robot.java

```diff
@@ -132,16 +137,32 @@ public void testPeriodic() {
 	}
 
 	public void autonomousPeriodic() {
-		autonomous.run();
+//		autonomous.run();
 	}
 
 	/**
 	 * This function is called periodically during operator control
 	 */
 	public void teleopPeriodic() {
+		//TODO: Set Min_DRIVE_SPEED in Robot Map.
+		double MIN_DRIVE_SPEED = 0.1;
+		driverstation.readInputs();
+		XboxController xbox = new XboxController(0);
+		double left = driverstation.getDriveJoystick().getLeftStickY()*-1;
+		double right = driverstation.getDriveJoystick().getRightStickX();
+		// -1* driverstation.getDriveJoystick().getJoystick()
+		LOGGER.info("left " + left + " right " + right) ;
+
+		if (Math.abs(left) < MIN_DRIVE_SPEED) {
+			left = 0.0;
+		}
+		if (Math.abs(right) < MIN_DRIVE_SPEED) {
+			right = 0.0;
+		}
```

Comment:
Also an important thing, although DifferentialDrive has a dead zone setting built in.